src/test/scala/com/gu/support/workers/IsAccountAccessAllowedSpec.scala

```diff
@@ -4,13 +4,13 @@ import com.gu.support.workers.model.AccountAccessScope.{SessionAccess, Authentic
 import com.gu.zuora.GetAccountForIdentity.{CreatedInSession, NotCreatedInSession}
 import org.scalatest.{FlatSpec, Matchers}
 
-class GetRecurringSubscriptionIsInAccountAccessScopeSpec extends FlatSpec with Matchers {
+class IsAccountAccessAllowedSpec extends FlatSpec with Matchers {
```

Comment:
given that half the idea of extracting the function was to make the test more reasonable, I should have remembered to update the test name to match!